examples/typescript/src/after.ts

```diff
@@ -0,0 +1,27 @@
+// TODO: stricter types and JSDoc 
+
+declare module '@jaredpalmer/after' {
+    import { RouteComponentProps } from 'react-router-dom';
+    // export interface AfterProps<P = {}> extends RouteComponentProps<P> {
+    //     prefetch: (pathname: string) => void;
+    //     refetch: (ctx: any) => void;
+    // }
+
+    export type AfterProps<P = {}> = RouteComponentProps<P> & P & {
```

Comment:
Not sure why `RouteComponentProps<P>` does not have members of `P`. This type doesn't seem right but it did unblock us